decidim-debates/spec/system/debates_dates.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Date comprober", versioning: true, type: :system do
+  include_context "with a component"
+  let(:manifest_name) { "debates" }
+  
+  let!(:debate) { create(:debate, component: component) }
+
+  before do
+    visit_component
+    click_link debate.title[I18n.locale.to_s], class: "card__link"
+  end
+
+  context "when shows the debate component" do
+    it "date same day" do
+	    Date.start_time.should be_between(Date.start_time.strftime("%h"),Date.end_time.strftime("%h"))
+    end
```

Comment:
Where is this `Date` coming from?  😮  What is the output of this test?

Usually, in system specs for decidim (which uses Capybara), things are checked like this:

`expect(page).to have_content "foo bar"`

https://github.com/decidim/decidim/blob/release/0.23-stable/decidim-debates/spec/system/show_spec.rb

You should write specs for all the different cases: For different `start_time` and `end_time` combinations, check that the view has the content that you expect. More feedback coming when you do so! 🥳